app/electron/runCmd.ts

```diff
@@ -166,3 +168,32 @@ export function handleRunCommand(
     event.sender.send('command-exit', eventData.id, code);
   });
 }
+
+/**
+ * Runs a script, using the compiled app, or Electron in dev mode.
+ *
+ * This is needed to run the "scriptjs" commands, as a way of running
+ * node js scripts without requiring node to also be installed.
+ */
+export function runScript() {
+  // If '..' in process.argv[1] or it starts with / or \, then it is not a valid path.
+  if (
+    !process.argv[1] ||
+    process.argv[1].includes('..') ||
+    process.argv[1].startsWith('/') ||
+    process.argv[1].startsWith('\\')
```

Comment:
If plugins can ship their own script, I am not sure what this will prevent.

Reply:
This limits it to running ones inside the plugins folder as a defensive measure.